README.md

```diff
@@ -345,6 +345,23 @@ Add the following to your project's pom.xml.
     </dependencyManagement>
 ```
 
+## Container Deployment
```

Comment:
For better visibility refer this: https://github.com/rahulKQL/java-bigtable/blob/78c5b04680073750f523962cdc41cf566c304b9a/README.md#container-deployment